Algorithm/BinaryTree/leetcode/110. Balanced Binary Tree/claude sonnet 4.6 adaptive/README_react.html

```
@@ -0,0 +1,1958 @@
+                        <li>ベースケース：node が None なら高さ 0 を返す</li>
+                        <li>左右を再帰し、-1 が返ってきたら即 -1 を伝播（早期リターン）</li>
+                        <li>
+                            左右の高さの差 &gt; 1 なら -1、そうでなければ max(左, 右) + 1 を返す
+                        </li>
+                    </ol>
+                </div>
+
+                <pre class="line-numbers"><code class="language-python">from typing import Optional
+
+class Solution:
+    def isBalanced(self, root: Optional[TreeNode]) -> bool:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)